data-upgrade-processes-permissions/src/main/java/org/exoplatform/migration/ProcessesPermissionsMigration.java

```diff
@@ -95,6 +96,7 @@ public void processUpgrade(String oldVersion, String newVersion) {
           workflowEntity.setManager(managers);
           workflowEntity.setParticipator(participators);
           updatedWorkflows.add(workflowEntity);
+          shouldUpgrade = true;
```

Comment:
no need to set this property each iteration, let's do it when we find that the list updatedWorkflows is empty